assets/elasticsearch/main.py

```
@@ -0,0 +1,86 @@
+        # temporary solution
+        self.docker_execute(f"bash -c '{ASSETS_DIR}/launch.sh'")
+        #time.sleep(20)
+        #self.docker_execute(f"bash -c '{ASSETS_DIR}/launch.sh'", check=False)
+        time.sleep(10)
+
+    def ingest(self):
+        """
+        Ingests the dataset at self.datasets_path
+        """
+        self.docker_execute([
+            f"python3 {ASSETS_DIR}/ingest.py {self.datasets_path}"
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)